README.md

```diff
@@ -17,6 +17,11 @@ Infrastructure
 * [ ] Set HSTS to 31536000 (1 year)
   * `strict-transport-security: max-age=31536000`
   * [ ] If the service is not hosted under `services.mozilla.com`, it must be manually added to [Firefox's preloaded pins](https://dxr.mozilla.org/mozilla-central/source/security/manager/tools/PreloadedHPKPins.json#184). This only applies to production services, not short-lived experiments.
+* [ ] Correctly set client IP
+  * [ ] Sanitize [X-Forwarded-For](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For) sent from the client. AWS and GCP's load balancers will do this automatically
+  * [ ] Make sure the web server and the application get the true client IP by configuring trusted IP's within Nginx or Apache
```

Comment:
is there anything additional to do here to also make sure this real ip shows up in the logs? or is that automagic with the default logging format and a correct implementation of the realip module?

Reply:
Both the nginx and apache modules actually overwrite the client ip used within the "connection metadata" (my term, not theirs), which is what they use to within the access logs. So it's essentially automagic, would only be borked by borking the access log format.